prime-dotnet-webapi/PrimeConstants.cs

```diff
@@ -23,6 +23,13 @@ public static class PrimeConstants
         public readonly static string PHARMANET_SSL_CERT_FILENAME = Environment.GetEnvironmentVariable("PHARMANET_SSL_CERT_FILENAME");
         public readonly static string PHARMANET_SSL_CERT_PASSWORD = Environment.GetEnvironmentVariable("PHARMANET_SSL_CERT_PASSWORD");
 
+        // CHES Email API
+        public readonly static string USE_CHES = Environment.GetEnvironmentVariable("useCHES") ?? "true";
```

Comment:
These constants dont have to be a string, they can be a bool. mail server port is an int, for example.
The default should probably be false, as well.
how about:

public readonly static bool CHES_ENABLED = bool.Parse(Environment.GetEnvironmentVariable("CHES_ENABLED") ?? "false");

Reply:
yes the default will be false as soon as testing is done